sdk/python/packages/flet-core/src/flet_core/colors.py

```diff
@@ -356,3 +356,14 @@
 DEEP_ORANGE_ACCENT_200 = "deeporangeaccent200"
 DEEP_ORANGE_ACCENT_400 = "deeporangeaccent400"
 DEEP_ORANGE_ACCENT_700 = "deeporangeaccent700"
+GREY = "grey"
```

Comment:
Please let me know when you add colors. (for my colors browser)
I added this "grey" color right now.
Did flutter 3.7 bring in some other new ones?

Reply:
I don't think it was introduced in 3.7 - looks like a missed it in the first place ☺️